src/components/table/table.module.scss

```
@@ -11,39 +14,124 @@ $HEIGHT-LARGE: 80px;
   * {
     box-sizing: border-box;
   }
+
+  &.fixed-header {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    overflow: hidden;
+
+    &.horizontally-scrollable-container {
+      overflow-x: auto;
+      overflow-y: auto;
+    }
+  }
 }
 
 .table-header {
-  display: flex;
+  display: grid;
   width: 100%;
   height: 32px;
   align-items: center;
-  position: relative;
+
+  &.sticky-header {
+    position: sticky;
+    padding: 0 3px;
+    top: 0;
+    left: 0;
+    z-index: 5;
+    background-color: var(--rp-ui-base-bg-000);
+    border-bottom: 1px solid var(--rp-ui-base-e-100);
+    flex-shrink: 0;
+    width: 100%;
+    box-sizing: border-box;
+  }
+
+  &.horizontally-scrollable {
+    overflow-x: auto;
+    min-width: fit-content;
+    padding: 0 3px;
+
+    &.sticky-header {
+      overflow-x: visible;
+      position: sticky;
+      top: 0;
+      left: 0;
+      z-index: 2;
+      background-color: var(--rp-ui-base-bg-000);
+      border-bottom: 1px solid var(--rp-ui-base-e-100);
+      flex-shrink: 0;
+      height: auto;
+      min-height: 32px;
+      overflow: visible;
+    }
```

I assume, we can skip properties that already exist in the `sticky-header` class. And also `overflow` is duplicated here. And `z-index` is changed, do we really need this change?
```suggestion
    &.sticky-header {
      overflow-x: visible;
      height: auto;
      min-height: 32px;
    }
```